canonical-websites/cn.ubuntu.com
# Update run script to v2.7.1

## Done
Update run script to v2.7.0. Replaced links to be relative.

## QA
- Run `./run clean`
- Run `./run`
- Go to http://0.0.0.0:8010/ and see it all works
